app/controllers/shelters_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SheltersController < ApplicationController
+  def index
+    @shelters = Shelter.all
+  end
+
+  def show
+    @shelter = Shelter.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    Shelter.create(shelter_params)
+    redirect_to "/shelters"
+  end
+
+  private
+  def shelter_params
```

Comment:
helper method for create action, extracts params wanted for new shelters created